spec/models/notification_type_spec.rb

```diff
@@ -44,4 +44,11 @@
       end
     end
   end
+
+  describe "#enabled?" do
+    it "detects properly" do
+      expect(FactoryGirl.create(:notification_type, :audience => NotificationType::AUDIENCE_USER)).to be_enabled
```

Comment:
Does these tests need to `create`?
How is `enabled?` related?  Should it instead test `instance.valid?`?
If not, should the `enabled?` tests test against all of the audiences?